### DISCUSSION TITLE
Sharing the futurecoder project using Pyodide and React

### DISCUSSION THREAD

**Discussion by alexmojaki** [xhlulu/react-pyodide-template#4] [Show and tell]
Just wanted to share my project http://futurecoder.io/

It uses Pyodide and React, thought you might be interested.

**Comment by xhluca** [comment]
Looks amazing! Thanks for sharing :)

**Reply by xhluca** [reply]
FYI I have moved this issue to Discussions as a "Show and Tell" post.
